### ISSUE TITLE
Wrong negative expectation with once

### ISSUE THREAD

**Issue by SergiyYarinovskiy** [rspec/rspec-mocks#1210]
Hello. I have such tests:
```ruby
  class A; end
  it 'calls new' do
    expect(A).to receive(:new).once
    A.new
  end
```
It finishes successfully. Next one:
```ruby
  class A; end
  it 'calls new' do
    expect(A).not_to receive(:new)
    # A.new
  end
```
Also fine. But if I combine `.not_to` with `.once` it behaves like it expects **to receive**:
```ruby
  class A; end
  it 'calls new' do
    expect(A).not_to receive(:new).once
    # A.new
  end
```
```
     Failure/Error: expect(A).not_to receive(:new).once
     
       (A (class)).new(*(any args))
           expected: 1 time with any arguments
           received: 0 times with any arguments
```
I am not sure if this is a bug. I would expect that `.once` will be ignored and last test passes
`rspec-mocks (3.7.0)`

Thank you

**Comment by myronmarston** [comment]
Thanks for reporting.  I'd consider this a bug.  I don't think we should support receive counts (e.g. `once`, `twice`, at_least(3).times`, etc) with `not_to receive`.  Want to take a stab at a fix?

**Comment by SergiyYarinovskiy** [comment]
@myronmarston OK, I'll investigate :slightly_smiling_face:

**Comment by SergiyYarinovskiy** [comment]
Since I don't have access to create branch in this repo, I forked it and created branch in my repo called `issue-1210`. Here is my fix proposal - https://github.com/SergiyYarinovskiy/rspec-mocks/commit/6420f56bfb9befc5f87a670bfce255eb1c3ad2ed
What is the process to create PR here?
Thanks

**Comment by myronmarston** [comment]
Pushing a branch to your fork is the right way to do it.  You just need to open the PR here:

https://github.com/rspec/rspec-mocks/compare/master...SergiyYarinovskiy:issue-1210

**Comment by mrageh** [comment]
I think this issue can was resolved in #1212, let's close it

**Comment by myronmarston** [comment]
Yep, closing.
